### PR TITLE
MTP-394 Fix Status li spacing and icon alignment

### PR DESCRIPTION
**Pull request by petterkoding**
Icon should align to first line of text

### CODE REVIEW

File: mt-kit/core/css/src/components/_status.scss [hunk 1]
```diff
@@ -15,10 +15,12 @@
     grid-area: heading;
     overflow-wrap: anywhere;
   }
+
   .important {
     grid-area: icon;
     display: flex;
-    align-items: center;
+    align-items: flex-start;
+    font-size: clamp(1.5869573348rem, 4.7816357022vw + 1rem, 2.368593037rem);
```
Comment (amundlindstol) [inline mt-kit/core/css/src/components/_status.scss:23]: use variable

**Review by amundlindstol — CHANGES_REQUESTED** [review]

File: mt-kit/core/css/src/components/_status.scss [hunk 2]
```diff
@@ -28,21 +30,34 @@
...
   .text {
     grid-area: text;
+
     :first-child:not(h2):not(h3):not(h4) {
       margin-top: var(--spacer-x-small);
     }
+
+    ul {
+      li {
+        margin-top: var(--spacer-xxx-small);
```
[the remark this answers is not in the document]
Comment (petterkoding) [reply mt-kit/core/css/src/components/_status.scss:49]: Just trying to adjust spacing. Designer wants the spacing between each `li` to be smaller. Its currently using inherited styles for margin-top:  `--spacer-xx-small`

**Review by tajakobsen — APPROVED** [review]
Looks good. :+1:

File: mt-kit/core/svelte/src/lib/svelte/content/Status.svelte [hunk 3]
```diff
@@ -17,8 +17,9 @@
 </script>
 
 <div class="status {className}">
-  <span class={statusType} data-testid="status-type" />
-  <slot name="heading" />
+  <span class="{statusType} h2" data-testid="status-type"/>
```
Comment (lha047) [inline mt-kit/core/svelte/src/lib/svelte/content/Status.svelte:20]: Looks like there is somthing about spacing. There should be a space between `" />` at the end

**Review by lha047 — CHANGES_REQUESTED** [review]
Looks good :D 

See comment about Prettier

**Review by tajakobsen — APPROVED** [review]
Looks good (again)

**Pull request merged**